### ISSUE TITLE
Start values are set to NaN after Calculate Scale Divisor

### ISSUE THREAD

**Issue by UlrichSi** [Open-Systems-Pharmacology/MoBi#194]
When "Calculate Scale Divisor" is used after simulation, the Start values of the molecules appear to be set to NaN.

Use the (internal) example project provided in Mantis, then:
1. Open the simulation.
2. Press "F5" to simulate.
3. Click on "Run&Analyze"/"Calculate Scale Divisor", then "Calculate", then "Ok".
4. Expand the simulation's configuration in the simulations browser. The MSV building block is flagged as different. Right-click it, then select "Show differences".

The differences list not only the changed scale divisors, but also the start values. If these are committed, they will show up in the MSV BB. 
No current workaround, either divisors or start values have to be entered manually in MSV, since a "Refresh from source" will both override concentrations and divisors.

**Comment by msevestre** [comment]
Hum was not aware of this issue. Going to look at that and see if we can squeeze it in upcoming 7.3

**Comment by msevestre** [comment]
I can reproduce exactly and only with the observations defined above
A concentration based project AND start value constant

@UlrichSi Well done.

**Comment by msevestre** [comment]
ok I'd like to discuss with @Yuri05 the actual implementation of Scale Divisir in case of concentration-based project. I feel that this might not be quite right but I might be mistaken

**Comment by msevestre** [comment]
All: In case of concentration project, shouldn't the scale factor be calculated for concentration outputs instead of amount?

**Comment by UlrichSi** [comment]
I guess the scale divisors should be used on the number that is calculated for a molecule container. If internally amount is used (even in a concentration-based project), then it should be calculated for amount. The intention is to avoid numerical underflow or overflow.

**Comment by StephanSchaller** [comment]
Yes, it depends how the models are handled internally / by the solver: do amount and conc. dependent projects look the same for the solver (and SFs are integrated in the same way for both) but are just displayed differently on the user interface?

**Comment by msevestre** [comment]
@all: Yeah ok. Then it's just a Mobi bug then :)

**Comment by StephanSchaller** [comment]
> Yeah ok. Then it's just a Mobi bug then :)

Yes...HIGH PRIORITY!

**Comment by msevestre** [comment]
nope

**Comment by StephanSchaller** [comment]
MoBi is all the headstart we have on other PBPK software... don't start neglecting it!
